_posts/2020-08-28-accesibility-in-mind.md

```diff
@@ -0,0 +1,74 @@
+Furthermore, WCAG provides rules about content visibility (for instance [1.4.4 - Resize text:](https://www.w3.org/TR/WCAG21/#resize-text) *“text can be resized without assistive technology up to 200 percent without loss of content or functionality”* and [1.4.10 - Reflow](https://www.w3.org/TR/WCAG21/#reflow), which sets a screen limit for scrolling in two dimensions). Following these guidelines provides advanced UX experience for users who do not really need accessibility, but they choose to zoom the app through browser settings or work on a small window or on a mobile device.
+
+## 6. It future-proofs
+
+Following the WCAG may benefit your web app in unexpected ways. Guideline [1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=132%2C211%2C311%2C325%2C131#info-and-relationships) suggests techniques G115 (Using semantic elements to mark up structure) and H49 (Using semantic markup to mark emphasized or special text). Why prefer `<header>`, `<section>` and `<article>` over good old `<div>` and `<span>`? There may not be a compelling reason today, but chances are there will be one tomorrow! Advancements in AI and SEO technology indicate that web crawlers will be able to reason about web content some day. Semantic HTML is standardized and supported by all browsers so why not prepare for the future today?
+
+## 7. It eliminates whole classes of UI bugs
+
+WCAG recommends semantic markup (see above), which helps clear up content for the developer. By imbuing HTML containers with meaning, the developer reduces cognitive load, which is known to be a major source of bugs.
+The benefits don't end with content-related bugs, though. Semantic markup is good medicine against *divitis* (i.e. overuse of `<div>` elements). The default CSS attributes of the various semantically-rich elements eliminate `block / inline` misconfiguration bugs – a major symptom of divitis.
+
+## 8. It improves efficiency
+
+Admittedly, efficiency should be at the top of the UX considerations list. Take for example productivity apps. These thrive on providing users with means of building "muscle memory" on the keyboard. Consequently, ease of use for each action is evaluated on a basis of number, complexity and intuitiveness of keystrokes required. These are all covered under rule [2.1 - Keyboard Accessible](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=132%2C211%2C311%2C325%2C131#keyboard-accessible). For builders of productivity apps, guideline 2.1 serves as a handy checklist.
```

Comment:
"Admittedly" is OK, but it sounds a bit like you're conceding a point. If you want to be more assertive (which would work well here, I think), you could just leave it out.

"on **the** basis of **the** number …" or just "evaluated based on the …"